cmd/helm/lint.go

```diff
@@ -92,15 +94,20 @@ func lintChart(path string) error {
 		return errLintNoChart
 	}
 
-	issues := lint.All(path)
+	linter := lint.All(path)
 
-	if len(issues) == 0 {
+	if len(linter.Messages) == 0 {
 		fmt.Println("Lint OK")
+		return nil
 	}
 
-	for _, i := range issues {
+	for _, i := range linter.Messages {
 		fmt.Printf("%s\n", i)
 	}
 
+	if linter.HighestSeverity == support.ErrorSev {
+		return errLintFailed
```

Comment:
@technosophos @michelleN: decided to just return the error, and let Cobra print it as `Error: lint failed`